connector/connect/README.md

```diff
@@ -48,56 +47,3 @@ export CONNECT_PLUGIN_EXEC_PATH=/path-to-protoc-gen-grpc-java-exe
 The user-defined `protoc` and `protoc-gen-grpc-java` binary files can be produced in the user's compilation environment by source code compilation, 
 for compilation steps, please refer to [protobuf](https://github.com/protocolbuffers/protobuf) and [grpc-java](https://github.com/grpc/grpc-java).
 
-
-### Run Spark Shell
-
-To run Spark Connect you locally built:
-
-```bash
-# Scala shell
-./bin/spark-shell \
-  --jars `ls connector/connect/target/**/spark-connect*SNAPSHOT.jar | paste -sd ',' -` \
-  --conf spark.plugins=org.apache.spark.sql.connect.SparkConnectPlugin
-
-# PySpark shell
-./bin/pyspark \
-  --jars `ls connector/connect/target/**/spark-connect*SNAPSHOT.jar | paste -sd ',' -` \
-  --conf spark.plugins=org.apache.spark.sql.connect.SparkConnectPlugin
-```
-
-To use the release version of Spark Connect:
-
-```bash
-./bin/spark-shell \
-  --packages org.apache.spark:spark-connect_2.12:3.4.0 \
-  --conf spark.plugins=org.apache.spark.sql.connect.SparkConnectPlugin
-```
-
-### Run Tests
-
-```bash
-# Run a single Python class.
-./python/run-tests --testnames 'pyspark.sql.tests.connect.test_connect_basic'
-```
-
-```bash
-# Run all Spark Connect Python tests as a module.
-./python/run-tests --module pyspark-connect --parallelism 1
-```
-
-
-## Development Topics
-
-### Generate proto generated files for the Python client
-1. Install `buf version 1.11.0`: https://docs.buf.build/installation
-2. Run `pip install grpcio==1.48.1 protobuf==3.19.5 mypy-protobuf==3.3.0 googleapis-common-protos==1.56.4 grpcio-status==1.48.1`
```

Comment:
This is already documented in https://spark.apache.org/docs/latest/api/python/development/contributing.html#environment-setup